appveyor.yml

```diff
@@ -1,5 +1,7 @@
 version: '{build}'
-image: Visual Studio 2019
+image:
+- Visual Studio 2019
+- Ubuntu

```

Comment:
Currently not running. We need to configure it to run build.sh instead of build.cmd

Reply:
Now it runs; I changed the image to Ubuntu1804, rather than latest, to avoid surprises